rs/protobuf/src/gen/registry/registry.subnet.v1.rs

```diff
@@ -3,6 +3,8 @@
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct SubnetRecord {
+    /// The the list of node_ids that represent the set of nodes
```

Comment:
nit: This comment doesn't originate from this PR (it just was placed differently), but it still has a typo: "The the". This could be, e.g., "The IDs of the nodes that are part of this subnet". I'd also be OK with leaving the status quo.